spring-cloud-azure-context/src/main/java/com/microsoft/azure/spring/cloud/context/core/AzureAdmin.java

```diff
@@ -144,7 +147,13 @@ public ServiceBusNamespace getOrCreateServiceBusNamespace(String namespace) {
     }
 
     public ServiceBusNamespace getServiceBusNamespace(String namespace) {
-        return azure.serviceBusNamespaces().getByResourceGroup(resourceGroup, namespace);
+        try {
+            return azure.serviceBusNamespaces().getByResourceGroup(resourceGroup, namespace);
+        } catch (NullPointerException ignore) {
+            // azure management api has no way to determine whether an service bus namespace exists
+            // Workaround for this is by catching NPE
+            return null;
```

Comment:
Do we need add any log here?

Reply:
Might not. NPE really means namespace not exist. So just returning null.